Bootstraps/Symfony.php

```diff
@@ -88,7 +89,7 @@ public function getApplication()
             $app->booted = true;
         }, $app);
 
-        if ($this->debug) {
+        if ($this->debug && ProcessSlave::$slave) {
```

Comment:
The tests don't use a PHP-PM slave and the `register_file` function failed miserably. Not sure if we should fix the function instead to check for the `$slave` property not being null